x-template.js

```diff
@@ -188,14 +188,14 @@ class Forgiving {
         (localName === 'style' || localName === 'script') &&
         node.textContent.includes(Forgiving.#CONTENT_MARKER)
       ) {
-        throw new Error(`Interpolation of "${localName}" tags is not allowed.`);
+        throw new Error(`Interpolation of <${localName}> tags is not allowed.`);
```

Comment:
This is the only functional change (and it’s minor). Just changes `"…"` characters to `<…>`.